WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragmentPage.kt

```diff
@@ -127,23 +123,23 @@ class NotificationsListFragmentPage : ViewPagerFragment(R.layout.notifications_l
         arguments?.let {
             tabPosition = it.getInt(KEY_TAB_POSITION, All.ordinal)
         }
-        notesAdapter = NotesAdapter( requireActivity(), this, null,
-            inlineActionEvents = viewModel.inlineActionEvents).apply {
-            this.setOnNoteClickListener(mOnNoteClickListener)
+        notesAdapter = NotesAdapter(requireActivity(), inlineActionEvents = viewModel.inlineActionEvents).apply {
+            onNoteClicked = { noteId -> handleNoteClick(noteId) }
+            onNotesLoaded = { itemCount -> updateEmptyLayouts(itemCount) }
             viewModel.inlineActionEvents.flowWithLifecycle(viewLifecycleOwner.lifecycle, Lifecycle.State.STARTED)
                 .onEach(::handleInlineActionEvent)
                 .launchIn(viewLifecycleOwner.lifecycleScope)
         }
         binding = NotificationsListFragmentPageBinding.bind(view).apply {
-            notificationsList.layoutManager = LinearLayoutManager(activity)
+            notificationsList.layoutManager = LinearLayoutManagerWrapper(activity)
```

Comment:
Passing the `context` should fix the CI build issues
```suggestion
            notificationsList.layoutManager = LinearLayoutManagerWrapper(view.context)
```